app/models/building.rb

```diff
@@ -4,6 +4,8 @@ class Building < ApplicationRecord
 
   validates :uprn, presence: true, uniqueness: true
 
+  enum on_delta: { No: 0, Yes: 1 }
```

Comment:
I'd use a native boolean type and remove this

Reply:
Have implemented this.